extensions/blocks/calendly/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import attributes from './attributes';
+import edit from './edit';
+import icon from './icon';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'calendly';
+export const title = __( 'Calendly', 'jetpack' );
+export const settings = {
+	title,
+
+	description: (
+		<Fragment>
+			<p>{ __( 'Embed a calendar for customers to schedule appointments', 'jetpack' ) }</p>
+		</Fragment>
+	),
```

Comment:
Do we need the `Fragment` and the paragraph tags here? It seems we should be able to do with just the string?